hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-timelineservice-hbase-tests/pom.xml

```diff
@@ -181,6 +181,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.inject</groupId>
+      <artifactId>guice</artifactId>
+      <version>${hbase-compatible-guice.version}</version>
```

Comment:
prior to this change, the module depends on guice version 4.2.3.
Guice was updated to 4.2.3 by HADOOP-17452.

Reply:
Correct, NodeManager and ResourceManager need to use guice that is compatible with hbase-compatible guava, hence this is one heck of a complex dependency chain.

I will also post my testing results for ATSv2 in 3.3.2 RC as well in a couple of days.